sporkfish/searcher/move_ordering/mvv_lva_heuristic.py

```diff
@@ -0,0 +1,46 @@
+import chess
+
+from ...board.board import Board
+from .move_order_heuristic import MoveOrderHeuristic
+
+
+class MvvLvaHeuristic(MoveOrderHeuristic):
+    # Columns: attacker P, N, B, R, Q, K
+    _MVV_LVA = [
```

Comment:
genuine q (not against): I thought I made this a protected attribute - and the test is able to access this matrix

Did you change it? If so, any particular reason?

Reply:
I made it a class level attribute - since we're initialising the MoveOrdering class a lot, we shouldn't have to initialise the matrix every time too.

Reply:
WOW - TIL! 

https://www.toptal.com/python/python-class-attributes-an-overly-thorough-guide